.github/workflows/mirror-php-package.yml

```diff
@@ -0,0 +1,68 @@
+name: Mirror PHP package
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        description: 'Enter package name to mirror (e.g. email-editor). It must refer to a package in the packages/php directory.'
+        required: true
+
+permissions: { }
+
+jobs:
+  mirror:
+    name: Run packages release script
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout monorepo
+        uses: actions/checkout@v4
+
+      - name: Check if package exists
+        id: check-package
+        run: |
+          if [ ! -d "packages/php/${{ github.event.inputs.package }}" ]; then
+            echo "Package ${{ github.event.inputs.package }} does not exist in packages/php"
+            exit 1
+          fi
+
+      - name: Setup mirror directory structure
+        run: |
+          # Create working directory for mirrors
+          mkdir -p mirror-working-dir
+          
+          # Create mirrors.txt file with target repository
+          echo "woocommerce/${{ github.event.inputs.package }}" > mirror-working-dir/mirrors.txt
+          
+          # Create subdirectory for the mirror repository
+          mkdir -p "mirror-working-dir/woocommerce/${{ github.event.inputs.package }}"
+          
+          # Copy package content to mirror directory
+          cp -r "packages/php/${{ github.event.inputs.package }}"/* "mirror-working-dir/woocommerce/${{ github.event.inputs.package }}/"
+          
+          # Copy any hidden files
+          cp -r "packages/php/${{ github.event.inputs.package }}/".* "mirror-working-dir/woocommerce/${{ github.event.inputs.package }}/" 2>/dev/null || true
+          
+      - name: Check if mirror prepare script exists
+        id: check-prepare-script
+        run: |
+          if [ -f "./packages/php/${{ github.event.inputs.package }}/tasks/mirror-prepare.sh" ]; then
+            echo "prepare_script_exists=true" >> $GITHUB_OUTPUT
+          else
+            echo "prepare_script_exists=false" >> $GITHUB_OUTPUT
+          fi
+  
+      - name: Cleanup mirror directory
+        if: steps.check-prepare-script.outputs.prepare_script_exists == 'true'
+        run: |
+          chmod +x "./packages/php/${{ github.event.inputs.package }}/tasks/mirror-prepare.sh"
+          ./packages/php/${{ github.event.inputs.package }}/tasks/mirror-prepare.sh "mirror-working-dir/woocommerce/${{ github.event.inputs.package }}"
+  
+      - name: Push to mirror
+        uses: Automattic/action-push-to-mirrors@v2.2.2
+        with:
+          token: ${{ secrets.API_TOKEN_GITHUB }}
+          username: matticbot
```

Comment:
I'm not sure about the credentials. I used the same as was in [the old mirrors workflow](https://github.com/woocommerce/woocommerce/pull/52613/files).